spec-tests/src/test/java/org/candlepin/spec/ExportSpecTest.java

```diff
@@ -193,13 +193,13 @@ public void shouldImportContentAccessCertsForAConsumerBelongingToOwnerInSCAMode(
 
         assertThat(caCerts).singleElement();
 
-        // Check if entitlement certs are present in exported zip file.
+        // Should not contain entitlement certificate in SCA mode
         List<ZipEntry> entitlementCerts = export.stream()
             .filter(entry -> entry.getName().startsWith(ENTITILEMENT_CERTIFICATES_PATH))
             .filter(entry -> entry.getName().lastIndexOf('/') == ENTITILEMENT_CERTIFICATES_PATH.length() - 1)
             .collect(Collectors.toList());
 
-        assertThat(entitlementCerts).singleElement();
+        assertThat(entitlementCerts).isEmpty();
```

Comment:
Double check with the Sat/Katello folks on this one. I'm pretty sure the manifests still need one or more of the certs for resyncing later. I forget which one off hand, so it'd be good to vet this with them to ensure we aren't about to break manifest import and resync.

Reply:
It is confusing, but what this test is calling isn't the 'manifest export' endpoint (`ConsumerResource.exportDataAsync`). It is the 'export certificates of a non-manifest consumer in a zip format' endpoint (`ConsumerResource.exportCertificates`), which isn't related to Satellite. Instead this is called from RHSM-Web/customer portal, where a 'disconnected' Hosted customer can go to the portal, create a normal (non-manifest consumer), add entitlements to it, and export the entitlement certs as a zip file. The reason the test is now getting an empty list of entitlements is because this consumer is non-manifest.

However, this endpoint should not longer be called if an org has been switched to SCA mode, because the user will be redirected to consoleDot, and never reach the customer portal page that would call this.
So I think what we need here is two versions of this test:
- The current one, where the org is in SCA mode, and no entitlement certs are exported
- Another where the org is in Entitlement mode, and entitlement certs are exported

Reply:
done, test added